server-provider/src/main/java/org/sonatype/tests/http/server/jetty/behaviour/Truncate.java

```diff
@@ -1,60 +0,0 @@
-/*
- * Copyright (c) 2010-2014 Sonatype, Inc. All rights reserved.
- *
- * This program is licensed to you under the Apache License Version 2.0,
- * and you may not use this file except in compliance with the Apache License Version 2.0.
- * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the Apache License Version 2.0 is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
- */
-package org.sonatype.tests.http.server.jetty.behaviour;
-
-import java.util.Map;
-
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-/**
- * @author Benjamin Hanzelmann
- */
-public class Truncate
```

Comment:
Jetty 9 doesn't like the content length being set to a value that was larger than the returned response. AFAICT we don't use this in the nexus 2 and 3 testsuite, so removing rather than spending time determining if we can implement this in an alternative way